modules/olm-cs-status-cli.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-status.adoc
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+ifdef::openshift-origin[]
+:global_ns: olm
+endif::[]
+ifndef::openshift-origin[]
+:global_ns: openshift-marketplace
+endif::[]
+
+[id="olm-cs-status-cli_{context}"]
+= Viewing Operator catalog source status using the CLI
+
+You can view the status of an Operator catalog source using the CLI.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the catalog sources for the namespace where your catalog source was created. For example, you can check the `{global_ns}` namespace, which is used for cluster-wide catalog sources:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get catalogsources -n {global_ns}
+----
++
+.Example output
+[source,terminal]
+----
+NAME                  DISPLAY               TYPE   PUBLISHER   AGE
+certified-operators   Certified Operators   grpc   Red Hat     55m
+community-operators   Community Operators   grpc   Red Hat     55m
+example-catalog       Example Catalog       grpc   Example Org 2m25s
+redhat-marketplace    Red Hat Marketplace   grpc   Red Hat     55m
+redhat-operators      Red Hat Operators     grpc   Red Hat     55m
+----
+
+. Use the `oc describe` command to get more details and status about a catalog source:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc describe catalogsource example-catalog -n {global_ns}
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+Name:         example-catalog
+Namespace:    {global_ns}
+...
+Status:
+  Connection State:
+    Address:              example-catalog.{global_ns}.svc:50051
+    Last Connect:         2021-09-09T17:07:35Z
+    Last Observed State:  TRANSIENT_FAILURE
+  Registry Service:
+    Created At:         2021-09-09T17:05:45Z
+    Port:               50051
+    Protocol:           grpc
+    Service Name:       example-catalog
+    Service Namespace:  {global_ns}
+----
++
+In this example output, the last observed state is `TRANSIENT_FAILURE`, which indicates that there is a problem establishing a connection for the catalog source.
```

Comment:
up to you on a split: "...`TRANSIENT_FAILURE`. This state indicates a problem..."